db/migrate/20221019134649_add_status_to_evidence_waiver_submission.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToEvidenceWaiverSubmission < ActiveRecord::Migration[6.1]
+  def change
+    add_column :claims_api_evidence_waiver_submissions, :status, :boolean
```

Comment:
why are we doing a boolean here? I thought we were going to use the same structure as the PoA model?

Reply:
that was a mistake, and a correction has been pushed